src/AaveV3ListGhoStewards_20230507/GhoAaveV3GhoSteward_20230507_Payload.sol

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {AaveV3Ethereum} from 'aave-address-book/AaveV3Ethereum.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {GhoSteward} from 'gho-core/contracts/misc/GhoSteward.sol';
+import {IGhoToken} from 'gho-core/contracts/gho/interfaces/IGhoToken.sol';
+
+/**
+ * @title Gho Stewards AIP
+ * @author @aave - AaveCompanies
+ * @dev This proposal enables GhoSteward for agile parameter changes in GHO. It consists of a set of actions:
+ * - Deployment of GhoSteward and transfers ownership to Aave DAO (short executor)
+ * - Grants role of PoolAdmin to GhoSteward
+ * - Grants role of BucketManager to GhoSteward
+ * Relevant governance links:
+ * - Governance: https://governance.aave.com/t/arfc-gho-steward-agile-parameter-changes/13922
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x67fa551993a94b801018d02154f3d4f27e29bea51fe6e862686de6bf9ee650af
+ */
+contract GhoAaveV3GhoSteward is IProposalGenericExecutor {
```

Comment:
You may need to add date here too, same name as the filename

Reply:
updated to in `GhoAaveV3GhoSteward_20230507_Payload` 3f169bdfd649faeca2ddbfbe92d00350ee5be7c5